go/runtime/history/db.go

```diff
@@ -149,48 +149,59 @@ func (d *DB) metadata() (*dbMetadata, error) {
 	return meta, nil
 }
 
-func (d *DB) commit(blk *roothash.AnnotatedBlock, roundResults *roothash.RoundResults) error {
+func (d *DB) commitBatch(blks []*roothash.AnnotatedBlock, results []*roothash.RoundResults) error {
```

Comment:
If you have only one method, you should think about renaming it to `commit`. Same holds for `History`, if you decide to remove `Commit`.